index.js

```diff
@@ -53,71 +208,104 @@ module.exports = {
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/consistent-function-scoping.md
+    'unicorn/consistent-function-scoping': 'off',
 
     // We shouldn't enforce filename casing to always be the same.  e.g. We may
     // prefer kebab-case for bin files, PascalCase for React components, etc.
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/filename-case.md
     'unicorn/filename-case': 'off',
 
-    // Uses safe-regex to disallow potentially catastrophic exponential-time
-    // regular expressions.
-    'unicorn/no-unsafe-regex': 'error',
+    // Nested ternaries are formatted well with Prettier and are commonly used in JSX
+    // https://eslint.org/docs/rules/no-nested-ternary
+    'no-nested-ternary': 'off',
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-nested-ternary.md
+    'unicorn/no-nested-ternary': 'off',
 
-    // Functions that take many positional arguments can be difficult to work
-    // with and produce less maintainable APIs. When more than three arguments
-    // are needed, named arguments should be used.
-    'max-params': ['error', 3],
+    // This rule is incompatible with React
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-null.md
+    'unicorn/no-null': 'off',
+
+    // Preventing abbreviations is too opinionated for linting and is not
+    // particularly compatible with React development
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/prevent-abbreviations.md
+    'unicorn/prevent-abbreviations': 'off',
+
+    // This rule is overly burdensome in and is premature optimization, at best
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/prefer-set-has.md
+    'unicorn/prefer-set-has': 'off',
 
     // When an anonymous function is the default export of a module, your stack traces
-    // and performance profiles become littered with Anoynmous functions which aren't
+    // and performance profiles become littered with anonymous functions which aren't
     // helpful with debugging.
-    'import/no-anonymous-default-export': ['error', {
-      allowArray: true,
-      allowLiteral: true,
-      allowObject: true,
-    }],
+    // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-anonymous-default-export.md
+    'import/no-anonymous-default-export': [
+      'error',
+      {
+        allowArray: true,
+        allowLiteral: true,
+        allowObject: true,
+      },
+    ],
+
+    // Disruptive to development and can be problematic with Redux
+    // Also, some people prefer named exports over default ones and there's
+    // nothing better or worse about each choice that should be forced
+    // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md
+    'import/prefer-default-export': 'off',
```

Comment:
Agreed, this is a rule that I often disable.  I don't think that a file with one export should be different than a file with two exports *from the consumer side*, and this rule causes that.